web/src/db.tsx

```diff
@@ -212,23 +254,42 @@ export interface WgsCovPlotDB {
   /** list of virus segment names if segmented virus being shown */
   segments?: string[];
 
+  /** Max length among samples of 1 segment**/
+  maxSegmentLength?: MaxSegmentLength
+
   /** Show amplicon features? */
-  show_amplicons: boolean;
+  show_amplicons?: boolean;
```

Comment:
Are there cases when `show_amplicons` will be null or undefined?

Same goes for `show_primer_matches` below.